core/admin/mailu/ui/views/domains.py

```diff
@@ -70,6 +70,27 @@ def domain_details(domain_name):
     domain = models.Domain.query.get(domain_name) or flask.abort(404)
     return flask.render_template('domain/details.html', domain=domain)
 
+@ui.route('/domain/details/<domain_name>/zonefile', methods=['GET'])
+@access.domain_admin(models.Domain, 'domain_name')
+def domain_download_zonefile(domain_name):
+    domain = models.Domain.query.get(domain_name) or flask.abort(404)
+    res = [domain.dns_mx, domain.dns_spf]
+    if domain.dkim_publickey:
+        record = domain.dns_dkim.split('"', 1)[0].strip()
+        txt = f'v=DKIM1; k=rsa; p={domain.dkim_publickey}'
+        txt = ' '.join(f'"{txt[p:p+250]}"' for p in range(0, len(txt), 250))
+        res.append(f'{record} {txt}"')
```

Comment:
I guess this is the part that must be modified. The part that introduces the unneeded extra double quote. When there is NO split, then use 
`res.append(f'{record} {txt}"')`
if there is a split, then use:
`res.append(f'{record} {txt}')`

Reply:
Yes and no. the quote in line 82 has to be removed for it to be correct. done.